docs/chat-configuration/ui-customization/how-it-works.md

```diff
@@ -7,8 +7,9 @@ grand_parent: Chat Configuration
 nav_order: 1
 ---
 
-# How it works {{site.data.vars.need-work}}
+# Customization of the chat UI {{site.data.vars.need-work}}
```

Comment:
what about:
"Customizing chat UI basics" 
since this doc go over the basics of the provided customization tools available.
each component has it's own twicks and changes from the basics which are elaborated on the component page.

Reply:
OK